installer/templates/phx_single/mix.exs

```diff
@@ -55,8 +55,8 @@ defmodule <%= @app_module %>.MixProject do
        app: false,
        compile: false,
        depth: 1},<% end %><%= if @mailer do %>
-      {:swoosh, "~> 1.5"},
-      {:finch, "~> 0.13"},<% end %>
+      {:swoosh, "~> 1.16"},
+      {:req, "~> 0.5.4"},<% end %>
```

Comment:
We can probably relax it a bit?

```suggestion
      {:req, "~> 0.5"},<% end %>
```

Reply:
I went for 0.5.x because I thought 0.6 may introduce breaking changes. Happy to go with either though.

Reply:
As you prefer! It is only for new apps and there will be a lock file anyway. And we can always update it.